test/performance/odp_queue_perf.c

```diff
@@ -608,25 +608,25 @@ int main(int argc, char **argv)
 
 	/* Init ODP before calling anything else */
 	if (odp_init_global(&instance, &init, NULL)) {
-		printf("Error: Global init failed.\n");
+		ODPH_ERR("Global init failed.\n");
 		return -1;
 	}
 
 	/* Init this thread */
 	if (odp_init_local(instance, ODP_THREAD_WORKER)) {
-		printf("Error: Local init failed.\n");
+		ODPH_ERR("Local init failed.\n");
 		return -1;
 	}
 
 	shm = odp_shm_reserve("queue_perf_global", sizeof(test_global_t), ODP_CACHE_LINE_SIZE, 0);
 	if (shm == ODP_SHM_INVALID) {
-		ODPH_ERR("Error: Shared mem reserve failed.\n");
+		ODPH_ERR("Shared mem reserve failed.\n");
 		exit(EXIT_FAILURE);
 	}
 
 	global = odp_shm_addr(shm);
 	if (global == NULL) {
-		ODPH_ERR("Error: Shared mem alloc failed\n");
+		ODPH_ERR("Shared mem alloc failed\n");
```

Comment:
Old code, but the "alloc" here for `odp_shm_addr()` is a bit confusing.

Reply:
Message clarified in V2.